Build failes for fullBeta flavor

The beta build has a different package name, which is not included in google-services.json.
Do we want to use crashalytics for beta builds? If so, @digitaldan, can you configure add new package id in crashalytics ?


````
:mobile:processFossBetaDebugGoogleServices
Parsing json file: ...\openhab.android\mobile\google-services.json
:mobile:processFossBetaDebugGoogleServices FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':mobile:processFossBetaDebugGoogleServices'.
> No matching client found for package name 'org.openhab.habdroid.beta'

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 5s
50 actionable tasks: 5 executed, 45 up-to-date

````

--- a/mobile/google-services.json
+++ b/mobile/google-services.json
@@ -15,6 +15,14 @@
       },
       "oauth_client": [
         {
+          "client_id": "849058498762-079qmadgd5f5hkdve7lkrbubfi842afj.apps.googleusercontent.com",
+          "client_type": 1,
+          "android_info": {
+            "package_name": "org.openhab.habdroid",
+            "certificate_hash": "28e3a41b7c84483ded3df7b8fc2789b5c912badd"
+          }
+        },
+        {
           "client_id": "849058498762-bc6nbdk0aercq1e5utj1so77e390cr5j.apps.googleusercontent.com",
           "client_type": 3
         }
@@ -29,8 +37,73 @@
           "status": 1
         },
         "appinvite_service": {
-          "status": 1,
-          "other_platform_oauth_client": []
+          "status": 2,
+          "other_platform_oauth_client": [
+            {
+              "client_id": "849058498762-bc6nbdk0aercq1e5utj1so77e390cr5j.apps.googleusercontent.com",
+              "client_type": 3
+            },
+            {
+              "client_id": "849058498762-3m681ei7b78ajqt9gmvt2gumah724v7q.apps.googleusercontent.com",
+              "client_type": 2,
+              "ios_info": {
+                "bundle_id": "es.spaphone.openhab",
+                "app_store_id": "492054521"
+              }
+            }
+          ]
+        },
+        "ads_service": {
+          "status": 2
+        }
+      }
+    },
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:849058498762:android:1e73d9e722953c8b",
+        "android_client_info": {
+          "package_name": "org.openhab.habdroid.beta"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "849058498762-b49s5qfketdcojkrue5mo1geh0k5q4pm.apps.googleusercontent.com",
+          "client_type": 1,
+          "android_info": {
+            "package_name": "org.openhab.habdroid.beta",
+            "certificate_hash": "28e3a41b7c84483ded3df7b8fc2789b5c912badd"
+          }
+        },
+        {
+          "client_id": "849058498762-bc6nbdk0aercq1e5utj1so77e390cr5j.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyBtollIxWyEIm3ni68CaoDJgZuUqoYC2Zw"
+        }
+      ],
+      "services": {
+        "analytics_service": {
+          "status": 1
+        },
+        "appinvite_service": {
+          "status": 2,
+          "other_platform_oauth_client": [
+            {
+              "client_id": "849058498762-bc6nbdk0aercq1e5utj1so77e390cr5j.apps.googleusercontent.com",
+              "client_type": 3
+            },
+            {
+              "client_id": "849058498762-3m681ei7b78ajqt9gmvt2gumah724v7q.apps.googleusercontent.com",
+              "client_type": 2,
+              "ios_info": {
+                "bundle_id": "es.spaphone.openhab",
+                "app_store_id": "492054521"
+              }
+            }
+          ]
         },
         "ads_service": {
           "status": 2

--- a/travis/prepare-deploy.sh
+++ b/travis/prepare-deploy.sh
@@ -12,8 +12,6 @@ then
     echo "Beta"
     releaseFlavor="beta"
     releaseFlavorCapital="Beta"
-    # Workaround for #794: Remove crashalytics
-    sed -i -e 's|maven.fabric.io/public|repo1.maven.org/maven2|' -e /fabric/d -e '/google-services/d' mobile/build.gradle
 fi
 echo "Build apk"
 time ./gradlew :mobile:assembleFull${releaseFlavorCapital}Release

